package.py

```diff
@@ -194,6 +194,6 @@ def package(redcap_zip="."):
     env.package_name = '%s-%s.tgz' % (env.project_name, redcap_version_and_package_type)
     cwd = os.getcwd()
     # create the package
-    local("cd %s && tar -cz --exclude='.DS_Store' \
+    local("cd %s && tar -cz --no-xattrs --exclude='.DS_Store' \
```

Comment:
BSD utils have such odd schisms with GNU that I am forced to make this useless comment whining about them.

`--no-xattrs` is the default behavior for GNU tar with no exceptions; in BSD-land this is the default behavior _if you're a non-root user using a desktop session_.

Excerpt from docs for this flag from [bsdtar's man page](https://www.freebsd.org/cgi/man.cgi?query=bsdtar&sektion=1&format=html).

>      --no-xattrs
>        (c, r, u, x modes only) Do not archive or extract extended file attributes.
>        This is the reverse of --xattrs and the default behavior if tar is run as non-root in x mode.